src/main/java/edu/kis/powp/jobs2d/drivers/adapter/MyAdapter.java

```diff
@@ -26,8 +27,8 @@ public void operateTo(int x, int y) {
 		ILine line = LineFactory.getBasicLine();
 		line.setStartCoordinates(this.startX, this.startY);
 		line.setEndCoordinates(x, y);
-
-		drawLine(line);
+		this.setPosition(x, y);
+		DrawerFeature.getDrawerController().drawLine(line);
```

Comment:
The name of the class should be fixed.
Inheritance is not used (no super drawLine calls). Should be removed!